src/ui-config/networksConfig.ts

```diff
@@ -279,6 +282,7 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
       name: 'Fantom Bridge',
       url: 'https://app.multichain.org/#/router',
     },
+    ratesHistoryApiUrl: 'https://aave-api-v2.aave.com/data/rates-history',
```

Comment:
All these could be the environment variable `process.env.NEXT_PUBLIC_SCREENING_URL` which I'll be renaming to `NEXT_PUBLIC_API_BASEURL` with the voters list: https://github.com/aave/interface/commit/d017cf9b1e1c93d5862ab60b0c7ebf381fee6891

You could update them now, or I'll clean them up in my voters list branch after merge, either way works.

Reply:
Since it makes sense to update the screening url as well and you've already made that change let's just wait to update in your PR